src/app/Scenes/HomeView/HomeView.tsx

```diff
@@ -75,6 +76,19 @@ export const HomeView: React.FC = () => {
     requestPushNotificationsPermission()
   }, [])
 
+  useEffect(() => {
+    const experiments = compact(queryData.homeView?.experiments)
+    experiments.forEach(({ name, variant, enabled }) => {
+      if (!enabled) {
+        console.warn(`Experiment is not enabled: ${name}`)
+      } else if (!variant) {
+        console.warn(`Experiment variant is missing for: ${name}`)
+      } else {
+        tracking.viewedExperiment(name, variant)
+      }
+    })
+  }, [queryData.homeView?.experiments])
```

Comment:
What do you think about moving this logic into a hook to keep the `HomeView` component concise and easier to read? The screen tracking in line 92 could go in there as well (or have its own hook).

I think it could be a good pattern, especially for the HomeView component, which could potentially become very large.

In the end, it could look like this:

```
  ...
```suggestion
  const savedArtworksCount = useSavedArtworksCount(data)
  useHomeViewExperimentTracking()
  useHomeViewScreenTracking()
  useHomeViewPrefetching()
  useHomeViewPermissions()
  ....
```

Reply:
I like that 👍🏽 

Refactored in https://github.com/artsy/eigen/pull/11048/commits/289ed0632be1f9b423dd3db45645b4b0f1ed48b3

I think we should move these new hooks into a new `src/app/Scenes/HomeView/hooks` directory, but I've kept that out this PR because it results in a much bigger diff, touching other files.